docs/guides/observability/axiom.md

```diff
@@ -77,6 +77,15 @@ stages:
 
 If you run into a "forbidden" error upon deployment, it is most often due to the Axiom API Token not having the correct permissions specified in this documentation. Go to the Axiom Dashboard and ensure your permissions match the ones listed in this documentation.
 
+### GitHub API rate limit exceeded
+
+> API rate limit exceeded for <IP_ADDRESS>. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.) - https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting``
+
+We are using the GitHub API to fetch the latest version of the Axiom AWS Lambda Layer.
```

Comment:
Totally a nit pick, but instead of `We are using`, you should use `The Serverless Framework uses`.